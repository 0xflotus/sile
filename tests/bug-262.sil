\begin[papersize=a5]{document}
\nofolios
\script[src=packages/lorem]
\hbox
\skip[height=43em] % change this to 45 for no bug symptoms
\lorem[words=80]
{
\set[parameter=document.lskip,value=4em]
This is an indented environment
}
\end{document}
