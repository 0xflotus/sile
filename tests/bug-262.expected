<tests/bug-262.sil>
Set paper size 	419.5275636	595.275597
Begin page
Mx 	8.3906
My 	11.9055
Mx 	28.3906
My 	454.9055
Set font 	Gentium;10;400;;normal;;LTR
T	76 79 82 69 77	(lorem)
Mx 	56.5801
T	73 80 83 85 77	(ipsum)
Mx 	85.6192
T	68 79 76 79 82	(dolor)
Mx 	111.3869
T	83 73 84	(sit)
Mx 	125.2405
T	65 77 69 84	(amet)
Mx 	149.7630
T	67 79 78	(con)
Mx 	164.6800
T	83 69	(se)
Mx 	173.1615
T	84 69	(te)
Mx 	181.2230
T	13	(-)
Mx 	8.3906
My 	467.9055
T	84 85 82	(tur)
Mx 	24.4014
T	83 65 68 73 80 83 67 73 78 71	(sadipscing)
Mx 	70.7100
T	69 76 73 84 82	(elitr)
Mx 	91.4620
T	83 69 68	(sed)
Mx 	108.4543
T	68 73 65 77	(diam)
Mx 	132.2971
T	78 79 78 85 77 89	(nonumy)
Mx 	169.9339
T	69 73 82	(eir)
Mx 	181.2230
T	13	(-)
Mx 	8.3906
My 	480.9055
T	77 79 68	(mod)
Mx 	30.4683
T	84 69 77 80 79 82	(tempor)
Mx 	64.6651
T	73 78 86 73 68 85 78 84	(invidunt)
Mx 	103.8082
T	85 84	(ut)
Mx 	116.3645
T	76 65 66 79 82 69	(labore)
Mx 	146.1668
T	69 84	(et)
Mx 	158.0443
T	68 79 76 79 82 69	(dolore)
Mx 	8.3906
My 	493.9055
T	77 65 71 78 65	(magna)
Mx 	38.5192
T	65 76 73 81 85 89 65 77	(aliquyam)
Mx 	79.0531
T	69 82 65 84	(erat)
Mx 	98.1857
T	83 69 68	(sed)
Mx 	114.3885
T	68 73 65 77	(diam)
Mx 	137.4420
T	86 79 76 85 80 84 85 65	(voluptua)
Mx 	176.5599
T	65 84	(at)
Mx 	8.3906
My 	506.9055
T	86 69 82 79	(vero)
Mx 	29.2677
T	69 79 83	(eos)
Mx 	45.1155
T	69 84	(et)
Mx 	55.5141
T	65 67 67 85 83 65 77	(accusam)
Mx 	92.9635
T	69 84	(et)
Mx 	103.3621
T	74 85 83 84 79	(justo)
Mx 	125.9189
T	68 85 79	(duo)
Mx 	143.7834
T	68 79 76 79 82 69 83	(dolores)
Mx 	176.5306
T	69 84	(et)
Mx 	8.3906
My 	519.9055
T	69 65	(ea)
Mx 	20.7051
T	82 69 66 85 77	(rebum)
Mx 	50.7979
T	83 84 69 84	(stet)
Mx 	69.2696
T	67 76 73 84 65	(clita)
Mx 	90.1974
T	75 65 83 68	(kasd)
Mx 	111.9553
T	71 85 66 69 82 71 82 69 78	(gubergren)
Mx 	157.8684
T	78 79	(no)
Mx 	171.5208
T	83 69 65	(sea)
Mx 	8.3906
My 	532.9055
T	84 65 75 73 77 65 84 65	(takimata)
Mx 	47.3612
T	83 65 78 67 84 85 83	(sanctus)
Mx 	80.8777
T	69 83 84	(est)
Mx 	95.3756
T	76 79 82 69 77	(lorem)
Mx 	122.3003
T	73 80 83 85 77	(ipsum)
Mx 	150.0746
T	68 79 76 79 82	(dolor)
Mx 	174.5775
T	83 73 84	(sit)
Mx 	8.3906
My 	545.9055
T	65 77 69 84	(amet)
Mx 	32.1593
T	76 79 82 69 77	(lorem)
Mx 	59.5951
T	73 80 83 85 77	(ipsum)
Mx 	87.8805
T	68 79 76 79 82	(dolor)
Mx 	112.8944
T	83 73 84	(sit)
Mx 	125.9942
T	65 77 69 84	(amet)
Mx 	149.7630
T	67 79 78	(con)
Mx 	164.6800
T	83 69	(se)
Mx 	173.1615
T	84 69	(te)
Mx 	181.2230
T	13	(-)
Mx 	8.3906
My 	558.9055
T	84 85 82	(tur)
Mx 	24.4014
T	83 65 68 73 80 83 67 73 78 71	(sadipscing)
Mx 	70.7100
T	69 76 73 84 82	(elitr)
Mx 	91.4620
T	83 69 68	(sed)
Mx 	108.4543
T	68 73 65 77	(diam)
Mx 	132.2971
T	78 79 78 85 77 89	(nonumy)
Mx 	169.9339
T	69 73 82	(eir)
Mx 	181.2230
T	13	(-)
Mx 	8.3906
My 	571.9055
T	77 79 68	(mod)
Mx 	30.4683
T	84 69 77 80 79 82	(tempor)
Mx 	64.6651
T	73 78 86 73 68 85 78 84	(invidunt)
Mx 	103.8082
T	85 84	(ut)
Mx 	116.3645
T	76 65 66 79 82 69	(labore)
Mx 	146.1668
T	69 84	(et)
Mx 	158.0443
T	68 79 76 79 82 69	(dolore)
Mx 	192.9827
My 	19.4739
T	77 65 71 78 65	(magna)
Mx 	223.2372
T	65 76 73 81 85 89 65 77	(aliquyam)
Mx 	263.8970
T	69 82 65 84	(erat)
Mx 	283.1555
T	83 69 68	(sed)
Mx 	299.4842
T	68 73 65 77	(diam)
Mx 	322.6636
T	86 79 76 85 80 84 85 65	(voluptua)
Mx 	361.9074
T	65 84	(at)
Mx 	192.9827
My 	31.4739
T	86 69 82 79	(vero)
Mx 	214.1790
T	69 79 83	(eos)
Mx 	230.3460
T	69 84	(et)
Mx 	241.0638
T	65 67 67 85 83 65 77	(accusam)
Mx 	278.8323
T	69 84	(et)
Mx 	252.9827
My 	43.4739
T	52 72 73 83	(This)
Mx 	273.2278
T	73 83	(is)
Mx 	282.4475
T	65 78	(an)
Mx 	295.2023
T	73 78 68 69 78 84 69 68	(indented)
Mx 	334.6760
T	69 78	(en)
Mx 	344.8127
T	86 73	(vi)
Mx 	352.4543
T	82 79 78	(ron)
Mx 	366.9611
T	13	(-)
Mx 	232.9827
My 	55.4739
T	77 69 78 84	(ment)
End page
Finish

